eureka-server/Dockerfile

```
@@ -0,0 +1,43 @@
+RUN sed -i 's/\r$//' gradlew && chmod +x gradlew
+
+RUN --mount=type=cache,target=/root/.gradle \
+    ./gradlew :eureka-server:dependencies --no-daemon
+
+COPY eureka-server/src eureka-server/src
+
+RUN --mount=type=cache,target=/root/.gradle \
+    ./gradlew :eureka-server:bootJar --no-daemon -x test
+
+FROM eclipse-temurin:21-jre-alpine
+RUN apk add --no-cache curl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)